task1.js

```diff
@@ -0,0 +1,4 @@
+process.stdin.on("data", (data) => {
+  data = data.toString().split("").reverse().join("");
+  process.stdout.write(data + "\n");
```

Comment:
Please format the output newline (From screenshot, no space between reversed string and new input):
![Screenshot 2022-12-11 205712](https://user-images.githubusercontent.com/8213940/206920565-8dca5d8f-8e9d-4dba-a5ee-431ebf4bf290.jpg)

Reply:
Thanks for your comment. Fixed. Now looks something like this
![image](https://user-images.githubusercontent.com/73526360/207443395-a26117a3-441c-4502-8feb-5c57be145c71.png)

Reply:
Thank you! Look good.